vendor/wp-now/src/wp-now.ts

```diff
@@ -449,14 +463,130 @@ export function getThemeTemplate( projectPath: string ) {
 	}
 }
 
-async function mountMuPlugins( php: PHP, vfsDocumentRoot: string ) {
-	await php.mount(
-		path.join( getWpNowPath(), 'mu-plugins' ),
-		// VFS paths always use forward / slashes so
-		// we can't use path.join() for them
-		createNodeFsMountHandler(
-			`${ vfsDocumentRoot }/wp-content/mu-plugins`
-		) as unknown as MountHandler
+async function mountInternalMuPlugins( php: PHP ) {
```

Comment:
I'm considering moving this function to a `mount.ts` file where we could later also add other mounts like the SQLite plugin.